x-pack/packages/kbn-elastic-assistant/impl/assistant/quick_prompts/quick_prompts.tsx

```diff
@@ -130,6 +130,7 @@ export const QuickPrompts: React.FC<QuickPromptsProps> = React.memo(
               >
                 <EuiBadge
                   color={badge.color}
+                  data-test-subj={`quickPrompt-${badge.name}`}
```

Comment:
Badge names can have spaces and other special characters -- this wouldn't be an issue here since only used for tests and not included in prod builds, right?

<img width="514" alt="image" src="https://github.com/user-attachments/assets/060b6b83-f2f6-4bea-b801-1cb74cb07e3c">

Reply:
that's right!